src/components/navigation/index.js

```diff
@@ -17,7 +17,10 @@ const Navigation = () => {
             <ul>
                 <li><NavLink to='/'><HomeIcon colour="#000046" /><p>Home</p></NavLink></li>
                 <li><NavLink to='/'><ProfileIcon /><p>Profile</p></NavLink></li>
-                <li><NavLink to='/'><CohortIcon /><p>Cohort</p></NavLink></li>
+                <li><NavLink to='/MyCohort'><CohortIcon /><p>Cohort</p></NavLink></li>
+
+                {/* Please use the link below to reach student search */}
```

Comment:
Remove comments if unneeded

Reply:
We left these comments for a reason so that the other teams know what to do with the link. Should we still remove it?